packages/account-v2/tailwind.config.ts

```diff
@@ -1,9 +1,17 @@
 import type { Config } from 'tailwindcss';
-import QuillTailwindConfig from '@deriv/quill-design/quill-tailwind/tailwind.config.cjs';
+const plugin = require('tailwindcss/plugin');
 
 export default {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
-    presets: [QuillTailwindConfig],
+    plugins: [
+        plugin(({ addUtilities }) => {
+            addUtilities({
+                '.d-none': {
```

Comment:
Doesn't `.hidden`  from tailwind class do the same?

Reply:
Yes !
Thanks for catching that